google/cloud/bigtable/table_admin_test.cc

```diff
@@ -207,7 +216,7 @@ TEST_F(TableAdminTest, LegacyConstructorSetsCustomCQ) {
 TEST_F(TableAdminTest, LegacyConstructorDefaultsPolicies) {
   auto admin_client = MakeAdminClient(kProjectId, TestOptions());
   auto admin = TableAdmin(std::move(admin_client), kInstanceId);
-  auto policies = TableAdminTester::Policies(admin);
+  auto policies = TableAdminTester::Opts(admin);
```

Comment:
Should we call these locals `options` too?

Reply:
Yes, I think so. Fixed.